app/models/flag.model.js

```diff
@@ -0,0 +1,18 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var FlagSchema = new Schema({
```

Comment:
You should refactor this model to be `listing-flag.model.js` since we know we will be adding other types of flags later on. If you do this, also make sure to change 'FlagSchema' to 'ListingFlagSchema', and change the mongoose model name to 'ListingFlag'